### DISCUSSION TITLE
dependency wack-a-mole from v5 to v6

### DISCUSSION THREAD

**Discussion by lifeiscontent** [remix-run/react-router#8993] [v5 to v6 Migration]
@ryanflorence When trying to upgrade from v5 to v6 using compat, I'm seeing warnings in the console from webpack saying:

```index.js?e2a7:556 [webpack-dev-server] WARNING in ./node_modules/react-router-dom-v5-compat/index.js 14:0-439
export 'Navigate' (reexported as 'Navigate') was not found in 'react-router' (possible exports: MemoryRouter, Prompt, Redirect, Route, Router, StaticRouter, Switch, __HistoryContext, __RouterContext, generatePath, matchPath, useHistory, useLocation, useParams, useRouteMatch, withRouter)
```

using these versions as dependencies:

```
    "history": "5.3.0",
    "react-router": "5.3.3",
    "react-router-dom": "5.3.3",
    "react-router-dom-v5-compat": "6.3.0",
```

any thoughts on how to move forward here?

**Comment by felipe-muner** [comment]
take a look here:

https://github.com/remix-run/react-router/discussions/8753

**Reply by lifeiscontent** [reply]
@felipe-muner I followed that, the issue is that the compat layer references react-router and the codebase we currently use also depends on react-router but an older version of it.

**Reply by felipe-muner** [reply]
Ohh, I just saw it...

I hope you fixed it

**Reply by lifeiscontent** [reply]
@felipe-muner nope, still stuck 😅
